### PR TITLE
CSS Two Column Layout First Draft

### PR DESCRIPTION
**Pull request by neldaza**
https://user-images.githubusercontent.com/88061673/136717346-df264281-4eb4-43ec-826c-b85c11bac83e.mov

### CODE REVIEW

**Comment by neldaza** [comment]
updated with adjustments!
[Screen Shot 2021-10-11 at 11 16 40 AM](https://user-images.githubusercontent.com/88061673/136836069-0bc2a364-308e-4815-966f-0df3896ea998.png)

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  width: 80%;
+  margin: auto;
+}
+
+div {
+  justify-content: center;
+}
+
+.heading {
+  display: flex;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:11]: If you want to use `display: flex` on an element, that's usually an indicator it should be a `row` in your layout. You can apply the additional styling here as a separate class in addition to `row`, but if you are trying to build a layout using proper "layout classes" `row` is your go-to tool for making a flexbox

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 50%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  padding: 6rem;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:20]: image

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+.column {
+  display: flex;
+  margin: -2rem;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:26]: image

File: css-two-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+.column-text {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:28]: Make sure you name `column` classes after their size, not what they will be applied to.

File: css-two-column-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  width: 45%;
+}
+
+img {
+  width: 75%;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:33]: Probably a good opportunity for a `column` class 🤔

**Comment by neldaza** [comment]
Screen Shot 2021-10-16 at 8 49 09 PM

**Pull request merged**